### DISCUSSION TITLE
Using the biome formatter with eslint

### DISCUSSION THREAD

**Discussion by jonasgroendahl** [biomejs/biome#1429] [Q&A]
Anyone who managed to get a setup with biome formatter and eslint for linting setup working? Some example code somewhere? :) i've a TypeScript project that I want to use the biome formatter on

**Comment by ValentinGurkov** [comment]
I'd also like to know. I'm wondering whether it is feasible (I'm sure it is possible) to use biome's listing and typescript-eslint for any rules not covered by biome yet. 
However, I'm not sure there would be any performance benefits when using them both.

**Reply by Conaclos** [reply]
We are working on a command to migrate from eslint #1752.

> However, I'm not sure there would be any performance benefits when using them both.

There is. Some big project use Biome and eslint in tandem and they reduced their linting time by disabling the rules available in Biome.

**Reply by ValentinGurkov** [reply]
Oh, great! I'll be on the lookout, since I like what's included in the "unreleased" section of the changelog!

**Comment by Conaclos** [comment]
> Anyone who managed to get a setup with biome formatter and eslint for linting setup working? Some example code somewhere? :) i've a TypeScript project that I want to use the biome formatter on

You cam simply use `biome format --write`.
If you wish also to organize your js imports, you can use `biome check --apply --linter-enabled=false`just `. You can also [disable the linter in the config file](https://biomejs.dev/reference/configuration/#linterenabled).

**Reply by ematipico** [reply]
I believe what OP is asking is to run the Biome formatter AND disable the eslint stylistic rules. Sort of `eslint-config-prettier`
